.travis.yml

```diff
@@ -34,9 +34,21 @@ cache: ccache
 # The default action for Travis-CI is to continue running even if a command fails.
 # See https://github.com/travis-ci/travis-ci/issues/1066.
 # Use the YAML block scalar header (|) to allow easier multiline script coding.
+# Install the desired CMake before running update_deps.py
```

Comment:
Nit: 3.10.2 is called out above but not here; keep consistent?

Reply:
I don't want to create a dependency where changing the version necessitates changing it in multiple places.  (It's more consistent if we *don't* duplicate.)